BraveRewardsExample/ViewController.swift

```diff
@@ -163,9 +173,9 @@ extension ViewController: RewardsDataSource {
     return url.host
   }
   
-  func retrieveFavicon(with url: URL, completion: @escaping (FaviconData?) -> Void) {
+  func retrieveFavicon(for pageURL: URL, faviconURL: URL?, completion: @escaping (FaviconData?) -> Void) {
```

Comment:
What I'm supposed to do here in `brave-ios`?
Get favicon by faviconURL, or by pageURL if faviconURL is nil?

Reply:
We can do either, I really only updated it to include both because it was confusing before which URL was sent and brave-ios could technically do both :) Now we have a choice